pkg/controllers/utils/test/events.go

```diff
@@ -0,0 +1,168 @@
+/*
+Copyright 2021.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package test
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	rbacv1 "k8s.io/api/rbac/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/watch"
+
+	cco "github.com/openshift/cloud-credential-operator/pkg/apis/cloudcredential/v1"
+
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	albv1aplha1 "github.com/openshift/aws-load-balancer-operator/api/v1alpha1"
+)
+
+// Event is a simplified representation of the watch event received from the controller runtime client.
```

Comment:
This test utility would need to be put somewhere to be added as a dependency as it's now used in more than 1 repository. Something for follow up PRs.

Reply:
Noted.